2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -214,13 +215,151 @@ Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina
+
++ gestione delle risorse: si occupa dell'amministrazione delle risorse, delle infrastrutture e dei servizi necessari per l'esecuzione dei processi di supporto;
+
++ automatizzazione dei processi: determina gli strumenti necessari per automatizzare i processi, come la compilazione dei file sorgenti e il sistema di versionamento automatico;
+
++ risoluzione dei problemi: affronta e risolve i problemi legati alla gestione dei processi.
+
+In sintesi, l'Amministratore è il motore che mantiene il progetto in movimento, assicurando che tutto funzioni senza intoppi.
+
+\ \
+===== Analista
+L'Analista svolge un ruolo estremamente delicato, facendosi carico di individuare i bisogni del proponente e di trasformarli in requisiti che saranno l'input delle attività successive. Il suo lavoro si svolge intensamente nel periodo di avvio del progetto, e si conclude con la stesura dell'Analisi dei Requisiti. \
+Il suo compito è di estrema rilevanza in quanto un'incompleta o superficiale analisi può impattare in modo sensibile sulle attività successive, causando ritardi e costi aggiuntivi, andando ad inficiare sulla qualità e completezza del prodotto finale. \
+
+Le responsabilità dell'Analista includono:
++ documento di Analisi dei Requisiti: l'Analista è incaricato di scrivere questo documento cruciale, che dettaglia i requisiti specifici del progetto.
+
++ interazione con i clienti: l'Analista lavora a stretto contatto con il Proponente o i Committenti per capire e studiare i loro bisogni;
+
++ classificazione dei requisiti: individua i requisiti e li classifica come funzionali e non funzionali, e come obbligatori, desiderabili o opzionali;
+
++ definizione del problema e degli obiettivi: l'Analista esamina la situazione attuale, identifica il problema e stabilisce gli obiettivi da raggiungere;
+
++ trasformazione dei bisogni in requisiti: durante l'Analisi dei Requisiti, l'Analista converte i bisogni dei clienti in requisiti specifici per la soluzione.
+
+\ \
+===== Progettista
+Il Progettista ha il compito di sviluppare una soluzione che soddisfi le esigenze identificate dall'Analista, rispettando i vincoli individuati. Il Progettista trasforma i requisiti in un'architettura che modella il problema. \
+
+Le responsabilità del Progettista includono:
+
++ sviluppo di un'architettura robusta e resistente ai malfunzionamenti;
+
++ realizzazione di soluzioni affidabili, efficienti, sostenibili e conformi ai requisiti;
+
++ decomposizione del sistema in componenti e organizzazione delle loro interazioni, ruoli e responsabilità;
+
++ garanzia di un basso grado di accoppiamento nell'architettura.
+
+\ \
+===== Programmatore
+Il Programmmatore è responsabile della realizzazione della soluzione attraverso la codifica. Il suo compito è realizzare quanto stabilito dalla progettazione, accompagnando implementazione e manutenzione del codice. \
+Il Programmatore è la figura più numerosa all'intero del gruppo con precise competenze tecniche.\
```

Comment:
Anche questa frase è in più nel contesto del PDCA

Reply:
La figura del programmatore è stata decisamente limitata nel contesto PDCA... ha senso tenerla in considerazione? Ho rimosso molte delle _frasi individuate_ dalla review.